src/main/java/com/nike/cerberus/client/auth/aws/StsCerberusCredentialsProvider.java

```diff
@@ -242,4 +244,8 @@ protected void authenticate() {
         credentials = new TokenCerberusCredentials(token.getClientToken());
         expireDateTime = expires;
     }
+
+    private boolean isChina(String regionName) {
+        return regionName.equals("cn-north-1") || regionName.equals("cn-northwest-1");
```

Comment:
Do you think just checking for "cn-" prefix would be sufficient in case more regions are added to AWS China?

Reply:
Another option that is more prescriptive would be to create a dictionary of China regions at the top of this file, and check if the region is in the dict. Then if more regions are added to China, it's easier to update the code.